src/Apps/Sell/Routes/PhotosRoute.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from "react"
+import { PhotosRoute_submission$key } from "__generated__/PhotosRoute_submission.graphql"
+import { Text } from "@artsy/palette"
+import { graphql, useFragment } from "react-relay"
+import * as Yup from "yup"
+import { Formik } from "formik"
+import { DevDebug } from "Apps/Sell/Components/DevDebug"
+import { SubmissionLayout } from "Apps/Sell/Components/SubmissionLayout"
+
+const FRAGMENT = graphql`
+  fragment PhotosRoute_submission on ConsignmentSubmission {
+    assets {
+      id
+      imageUrls
+    }
+  }
+`
+
+const Schema = Yup.object().shape({
+  assets: Yup.array(),
+  // .min(1)
+  // .transform(fields => fields.filter(c => !c.errorMessage))
+  // .of(Yup.object().test("photos", value => value.assetId)),
```

Comment:
(I imagine this is a future TODO)

Reply:
Yep, I didn't want to overload this PR with the photo uploading/associating logic but wanted to at least insert the step along with basic navigation+query support.